app/views/portal/portal/home.html.erb

```diff
@@ -1,7 +1,7 @@
-<% content_for :title, t(".title", name: current_client.intake.primary_full_name) %>
+<% content_for :title, t(".title", name: current_client.intake.preferred_name) %>
```

Comment:
This is an unrelated change to the purpose of the pull request, but we were in the neighborhood and wanted to do it.